member-service/src/main/java/com/hedvig/memberservice/commands/EditMemberInformationCommand.kt

```diff
@@ -0,0 +1,11 @@
+package com.hedvig.memberservice.commands
+
+import com.hedvig.memberservice.web.dto.InternalMember
+import org.axonframework.commandhandling.TargetAggregateIdentifier
+
+class EditMemberInformationCommand(
```

Comment:
You know this better than me, but don't we usually make commands into data classes?
```suggestion
data class EditMemberInformationCommand(
```